client/state/ui/actions.js

```diff
@@ -1,7 +1,11 @@
 /**
  * Internal dependencies
  */
-import { SET_SELECTED_SITE, SET_SECTION } from 'state/action-types';
+import {
+	SET_SELECTED_SITE,
+	SET_SECTION,
+	USER_SET_CURRENT
```

Comment:
>Would it be possible to update the other actions to the new naming convention in this PR or in another one closely after?

Totally agree. It's been on my To-Do list to update #1378 with a section on actions (structure, type naming, etc). At that point, probably also makes sense to link to said guidelines from `state/action-types.js` so that it's obvious where to find the current recommendations.

Before I merge this in the morning, I'll plan to open a separate pull request with revisions to legacy action names.